src/Compute/Compute/VirtualMachine/Config/AddAzureVMIPConfigurationCommand.cs

```diff
@@ -0,0 +1,326 @@
+﻿//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Warning: This code was generated by a tool.
+//
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Compute.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+
+namespace Microsoft.Azure.Commands.Compute
+{
+    [Cmdlet(VerbsCommon.Add, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VMIPConfiguration"), OutputType(typeof(VirtualMachineNetworkInterfaceIPConfiguration))]
+    public partial class AddAzureVMIPConfigurationCommand : Microsoft.Azure.Commands.ResourceManager.Common.AzureRMCmdlet
```

Comment:
Why choose AddIpConfig vs NewIpConfig -- NewIpConfig is what is used for VMSS, so some consistancy could be good.